### PR TITLE
kiosk: use button control from react common

### PR DESCRIPTION
**Pull request by eanders-ms**
Swap out kiosk's button control for react-common's.

### CODE REVIEW

File: kiosk/src/Components/GameMenu.tsx [hunk 1]
```diff
@@ -30,6 +28,7 @@ export const GameMenu: React.FC<IProps> = () => {
         <div className="gameMenu">
             {game?.userAdded && !kiosk.locked && (
                 <GenericButton
+                    title={lf("Delete Game")}
```
Comment (kimprice) [inline kiosk/src/Components/GameMenu.tsx:31]: Do you need a label here?

**Review by kimprice — APPROVED** [review]
Other than that missing label, looks good!

File: kiosk/src/Components/GameMenu.tsx (hunk 1 above)
Comment (eanders-ms) [reply kiosk/src/Components/GameMenu.tsx:31]: In this case button content is provided by the nested svg and other child elements, so we don't want to send a label.

File: kiosk/src/Components/AddingGame.tsx [hunk 2]
```diff
@@ -112,8 +114,12 @@ const AddingGame: React.FC<IProps> = ({}) => {
                 <div style={{ flexGrow: 1 }} />
                 {qrDivContent()}
             </div>
-            <GenericButton autofocus={true} onClick={gotoMainMenu}>
-                {lf("Return to menu")}
+            <GenericButton
```
[the remark this answers is not in the document]
Comment (eanders-ms) [reply kiosk/src/Components/AddingGame.tsx:117]: yep!

**Pull request merged**